app/modules/asset_groups/models.py

```diff
@@ -408,16 +409,24 @@ def detected(self, job_id, response):
         if job is None:
             raise HoustonException(log, f'job_id {job_id} not found')
 
-        status = response.get('status')
+        status = message.get('status')
```

Comment:
i may be wrong here, but i _feel like_ i may have seem some kind of error response where `status` was not set at all ... wonder if adding a default of `False` on this might help cushion that possibility.

Reply:
It will, which is why I chose to do it via the get :)